README.md

```diff
@@ -2,8 +2,12 @@
 
 [![Join the chat at https://gitter.im/mozillascience/software-citation-tools](https://badges.gitter.im/mozillascience/software-citation-tools.svg)](https://gitter.im/mozillascience/software-citation-tools?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-The Software Citation Tools will be a suite of tools for the scientific community to assist them in citing software that is found online and used in research and scientific papers.  The purpose of this is to reduce friction for researchers to cite software and make sure that software developers are given the credit they are due.  The suite will be developed in conjunction with Mozilla Science Lab via Abigail Cabunoc Mayes, the scientific community, and the open source software development community.  The project will require extensive interviews with members of the research community to gather information on how they are currently citing software, and how these tools can streamline that process.
+The Software Citation Tools is a package of tools written in javascript and made available through the npm javascript package manager. The main part of the package will be a library which will create a citation when given a link to a github repository. Further tools will be created on top of this library to make using this library easier and extend it's usefulness.
 
-At the core of the Software Citation Tools suite will be a package written in JavaScript using npm.  This package will provide functions which will take a GitHub repository URL and return a citation for the software compliant with the Software Citation Principles.  Stretch goals for this package may include accepting other types of software repositories as input, and accepting parameters that will change the format of the citation.  For example, parameters to generate long and short versions of a citation, or output for different citation standards.  The priority of stretch goals will be determined based on the needs of the scientific community.
+##About the Project
+The Software Citation Tools is a project proposed and overseen by Mozilla Science Lab via Abigaill Cabunoc Mayes and headed by senior project team at The Rochester Institute of Technology's Software Engineering Department. The purpose of Software Citation Tools is to provide some tools that use the ideas shown in the [Software Citation Principles](https://www.force11.org/software-citation-principles) paper by FORCE11 to make software citation easier and more cited in academia.
```

Comment:
More grammatical stuff:
- "The Software Citation Tools" should be "Software Citation Tools" (see comment on line 5)
- "Abigaill" should be "Abigail"
- "headed by senior project team" should be "headed by a senior project team" (unless you planned to replace this?)
- further down, on line 11, you capitalize Senior Project, but here you don't.  Either is fine by me, as long as we're consistent.
- in "to make software citation easier and more cited in academia", your subject is "software citation" but your intended subject for the second part of the clause is "software".  You could clear this up either by changing "cited" to "frequent", or by explicitly adding the second subject ( i.e. "and to make software more cited in academia").